CONTRIBUTING.md

```diff
@@ -8,25 +8,57 @@ Before opening an issue, please search the [issue tracker](https://github.com/re
 
 Please ask any general and implementation specific questions on [Stack Overflow with a Redux Toolkit tag](http://stackoverflow.com/questions/tagged/redux-toolkit?sort=votes&pageSize=50) for support.
 
-## New Features
+We ask you to do this because StackOverflow has a much better job at keeping popular questions visible. Unfortunately good answers get lost and outdated on GitHub.
 
-Please open an issue with a proposal for a new feature or refactoring before starting on the work. We don't want you to waste your efforts on a pull request that we won't want to accept.
+Some questions take a long time to get an answer. If your question gets closed or you don't get a reply on StackOverflow for longer than a few days, we encourage you to post an issue linking to your question. We will close your issue but this will give people watching the repo an opportunity to see your question and reply to it on StackOverflow if they know the answer.
+
+Please be considerate when doing this as this is not the primary purpose of the issue tracker.
```

Comment:
i don't agree with this - they should either open a discussion or try the reactiflux  #redux channel, not use the issue tracker

Reply:
@EskiMojo14 
Some questions take a long time to get an answer. If your question gets closed or you don't get a reply on StackOverflow for longer than a few days, we encourage you to either open a discussion or try the reactiflux (#redux channel) on discord our redux community will help you resolving the issue.
How does this sound?

Reply:
yeah, i think:
```suggestion
If your question gets closed or you don't get a reply after a few days, consider opening a [discussion](https://github.com/reduxjs/redux-toolkit/discussions) or joining the [Reactiflux](https://discord.gg/reactiflux) discord server and asking in the #redux channel.
```